### PR TITLE
Changed 'E-mail' to 'Email'

### LINKED ISSUE #2122: 'Email' or 'E-mail' wording
Hello,

I'm happy to make the changes to this, just wanted to check which one should be the consistent spelling of e(-)mail.

The menu item is 'Email':
image

The page that opens is 'E-mail':
image

The form for setting it up reverts back to 'Email':
image

I'm happy to change them either way, it should just be the same spelling each time.

Thanks

### PR DESCRIPTION
**Pull request by jonpwilson**
Fixes #2122

### CODE REVIEW

**Comment by phillxnet** [comment]
@jonpwilson It's a little late in my location but I've just had a quick look and this pull request shares a related issue the the one you closed, ie the fixtures stuff.

Fixtures are test data for unit test and if you change stuff in there it may well break tests that looks to see if what it expects is actually happening, by way of feed the code dummy data and then asking the code what it sees. So we can't change stuff in there without potentially seriously jeopardising the unit tests.

Related to these fixture changes, maybe, is the change you made in: test_email_client.py as this may use those fixtures to set it's expectation. But it's also paired with email_client.py, that is the test_email_client.py likely exercises the email_client.py. So those 2 have to be kept in sync re error messages as one will output a message and the other, possibly using face data from fixtures, exercise the code to provoke various reactions, hence the requirement that they be kept in sync.

So if you could revert those fixture changes or however you want to do it that would be dandy as this would be another great one to have sorted.

If it's easier you could just nuke and pave again and this time leave the fixtures files out from your editing process.

Sorry to have to to bring this news but it's potentially a tone of work to re-work the test data in fixtures and it's never actually displayed, they just do string comparisons to see if the code has worked as intended.

Hope that helps and well done getting all this done so quickly. Much appreciated.

**Comment by jonpwilson** [comment]
Hi @phillxnet,

No problem, I knew I was changing the tests (that's why I changed both email_client.py and test_email_client.py) so I did check against the changes against the tests running locally but looking through the other PRs i appears that I'm not running the full suite of tests for some reason (mine shows 25 tests run, while FroggyFlox's shows 189!) so I'm happy to pull back from the fixtures. 

I've reverted changes to anything that's not a template. Typically, I try to help and just end up creating more work for you! :)

File: src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst [hunk 1]
```diff
@@ -86,10 +86,10 @@
     <div class="modal-content">
       <div class="modal-header">
 	<button type="button" class="close" data-dismiss="modal" aria-hidden="true">x</button>
-	<h3 id="myModalLabel">Test e-mail sent successfully.</h3>
+	<h3 id="myModalLabel">Test email sent successfully.</h3>
       </div>
       <div class="modal-body">
-	<p>Test e-mail has been successfully queued to postfix. Your mail from <code>{{email.sender}}</code> to <code>{{email.receiver}}</code> could take up to several minutes to arrive. If you do not receive it, check <code>/var/log/maillog</code> for clues. You may want to verify the input, delete and add the E-mail account again.</p>
+	<p>Test email has been successfully queued to postfix. Your mail from <code>{{email.sender}}</code> to <code>{{email.receiver}}</code> could take up to several minutes to arrive. If you do not receive it, check <code>/var/log/maillog</code> for clues. You may want to verify the input, delete and add the E-mail account again.</p>
```
Comment (phillxnet) [inline src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst]:
```suggestion
	<p>Test email has been successfully queued to postfix. Your mail from <code>{{email.sender}}</code> to <code>{{email.receiver}}</code> could take up to several minutes to arrive. If you do not receive it, check <code>/var/log/maillog</code> for clues. You may want to verify the input, delete and add the email account again.</p>
```

File: src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst [hunk 2]
```diff
@@ -25,24 +25,24 @@
...
-	      Rockstor will login to this e-mail account and use it to send
-	      notifications to another specified e-mail address, such as your
+	    <p>Rockstor can push notifications to you via email. We recommend
+	      creating a separate dedicated email account for this purpose.
+	      Rockstor will login to this email account and use it to send
+	      notifications to another specified email address, such as your
 	      personal gmail account.</p>
 	  </div>
-      	<button type="Submit" id="Add-email-address" class="btn btn-primary" title="Add an e-mail account to be used for sending notifications.">Add an E-mail account</button>
+      	<button type="Submit" id="Add-email-address" class="btn btn-primary" title="Add an email account to be used for sending notifications.">Add an E-mail account</button>
```
Comment (phillxnet) [inline src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst]:
```suggestion
      	<button type="Submit" id="Add-email-address" class="btn btn-primary" title="Add an email account to be used for sending notifications.">Add an email account</button>
```

Comment (phillxnet) [reply src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst]: Remaining stray E-mail to email change suggestion?

File: src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst (hunk 1 above)
Comment (phillxnet) [reply src/rockstor/storageadmin/static/storageadmin/js/templates/email/email_setup.jst]: Another remaining stray E-mail to email change suggestion?

**Comment by phillxnet** [comment]
@jonpwilson Re:
> No problem, I knew I was changing the tests (that's why I changed both email_client.py and test_email_client.py)

Apologies, it's difficult to know folks knowledge with only, as yet, fleeting interactions within pull requests.

> I did check against the changes against the tests running locally but looking through the other PRs i appears that I'm not running the full suite of tests for some reason (mine shows 25 tests run, while FroggyFlox's shows 189!)

That is strange, the following wiki details the running of our unit tests on a source install so that might help to track down the 'missing' ones:

openSUSE dev notes and status: https://forum.rockstor.com/t/opensuse-dev-notes-and-status/5662

We could continue this test anomaly issue in the forum. You could open a forum thread on how you are currently running them maybe?

Thanks for stepping up to yet another language consistency issue. Much appreciated.

I've had a look at the revised changes here and bar a couple of suggestions this look great. If you agree with the suggested changes then use the commit changes button and we can get this one merged. Or if you think we should leave the 2 remaining "E-mail" instances then do say so and we can take it from there.

**Comment by jonpwilson** [comment]
> Apologies, it's difficult to know folks knowledge with only, as yet, fleeting interactions within pull requests.

No need to apologise, it's your project - I'm just some random who has come in with a load of quick fixes. If you want me to leave the testing out of it, then it's no problem for me - you know the project much better than I so. I'm just happy to give back to something that's helping me out!

> We could continue this test anomaly issue in the forum. You could open a forum thread on how you are currently running them maybe?

Brilliant, I'll have a read through and see where I've gone wrong first. 

> I've had a look at the revised changes here and bar a couple of suggestions this look great. If you agree with the suggested changes then use the commit changes button and we can get this one merged. Or if you think we should leave the 2 remaining "E-mail" instances then do say so and we can take it from there.

Spot on, I've added those - apologies for missing them.

Thanks again

**Comment by phillxnet** [comment]
@jonpwilson Super, have now merge these changes. Thanks again for your time and effort and these changes will be in the next rpm release: slated currently to be 3.9.2-53.

**Pull request merged**